test/remote/gateways/remote_cyber_source_rest_test.rb

```diff
@@ -146,13 +146,13 @@ def test_successful_capture_with_partial_amount
     assert_equal 'PENDING', response.message
   end
 
-  def test_failure_capture_with_higher_amount
-    authorize = @gateway.authorize(@amount, @visa_card, @options)
-    response = @gateway.capture(@amount + 10, authorize.authorization, @options)
+  # def test_failure_capture_with_higher_amount
```

Comment:
Could you add test to ensure we are sending the appropriate commerceIndicator fields for NTs when using stored credentials?

Reply:
I can't because NT is not merged into master. Once that is I can write tests